zoo-css-challenge/notes:

```diff
@@ -0,0 +1,5 @@
+notes:
+changing borders on bear images - could i add ids to each of the bear images and style them that way or would a better way be 
+to use a css selector like 'first-of-type' within the class of image-container?
```

Comment:
This exercise wants you to use `first-of-type`/`nth-child(2)` to practice using those kinds of selectors. 

As mentioned in class, software teams typically prefer using class selectors so they would be preferred over ID selectors 👍